MyLifeUITests/MyLifeUITests.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  MyLifeUITests.swift
+//  MyLifeUITests
+//
+//  Created by George Galan on 08/01/2019.
+//  Copyright © 2019 Apple Inc. All rights reserved.
+//
+
+import XCTest
+
+class MyLifeUITests: XCTestCase {
+
+    let app = XCUIApplication()
+
+    override func setUp() {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+
+        // In UI tests it is usually best to stop immediately when a failure occurs.
+        continueAfterFailure = false
+
+        // UI tests must launch the application that they test. Doing this in setup will make sure it happens for each test method.
+        app.launch()
+
+        // In UI tests it’s important to set the initial state - such as interface orientation - required for your tests before they run. The setUp method is a good place to do this.
+    }
+
+    override func tearDown() {
+        app.terminate()
+    }
+
+    func testExample1() {
+
+        let NavigationBar = app.navigationBars["My Life"]
+        let PersonNavigationBar = app.navigationBars["MyLife.PersonView"]
+        let tables = app.tables
+        let nameTextField = tables.textFields["Name"]
+        let name = "New User"
+
+        // Tap on Add a new person
+        NavigationBar.buttons["Add"].tap()
+        XCTAssertTrue(nameTextField.exists, "Wrong screen buddy")
```

Comment:
In general try to wait for the element to exist because in some cases might be loaded slower